client/html/pages/panel/index.html

```diff
@@ -4,50 +4,37 @@
       <div class="row">
         <div class="col-md-8">
 
-          {{#if isInRole 'student' 'user-type'}}
+          {{#if isStudent}}
             <h2>Bem vindo(a), {{ user.profile.nome }}!</h2>
-          {{/if}}
-
-          {{#if isInRole 'company' 'user-type'}}
+          {{else}}
             <h2>Bem vindo(a), {{ user.profile.dados }}!</h2>
           {{/if}}
 
-          {{#unless currentUser.emails.[0].verified}}
-            <h3>Você precisa confirmar o seu endereço de email antes de utilizar esse portal!</h3>
-            <h4>Verifique seu email {{user.emails.[0].address}} ou <a href="#" class="resend-verification-link">envie o link de verificação novamente</a></h4>
-          {{else}}
           <div id="activity" class="tab-pane">
             <div class="card card-underline no-margin">
               <div class="card-head">
 
-                {{#if isInRole 'student' 'user-type'}}
+                {{#if isStudent}}
                   <header>O que está acontecendo?</header>
-                {{/if}}
-
-                {{#if isInRole 'company' 'user-type'}}
+                {{else}}
```

Comment:
For now, as we don't have a third user, like an Admin, we don't need to think about "what else" but we should not forget that this might need a plus verification in the future :)